features/issue/components/issuesList/IssuesList.tsx

```diff
@@ -42,11 +67,11 @@ export const IssuesList = () => {
 		)
 	})
 
-	const itemsPerPage = 5
-	const lastItemIndex = currentPage * itemsPerPage
-	const firstItemIndex = lastItemIndex - itemsPerPage
-
-	const currentIssues = filteredIssues.slice(firstItemIndex, lastItemIndex)
+	const currentIssues = paginationHandler({
+		paginatedItems: filteredIssues,
```

Comment:
paginatedItems jako argument słabo siedzi. Imo to powinno być po prostu items, paginatedItems to bardziej wynik zwrotny funkcji.
paginationHandler też słaba nazwa, imo getPaginatedItems po prostu lepsze, funkcja raczej jako czasownik powinna się nazywać